libs/ui/src/lib/TextField.tsx

```diff
@@ -0,0 +1,136 @@
+import styled from '@emotion/styled';
+import React, {
+  forwardRef,
+  InputHTMLAttributes,
+  ReactNode,
+  Ref,
+  useState,
+} from 'react';
+import { ThemeProps } from '../color';
+import { useBoolean } from '../hooks/useBoolean';
+
+export type ValidationState = 'valid' | 'invalid';
+
+export interface Validation {
+  validationState?: ValidationState;
+  isRequired?: boolean;
```

Comment:
how about this?

```suggestion
required?: boolean
```

we need to convention!

i think, boolean props does not need to `is` prefix

like HTML Input Element.

```html
<input required /> // not isRequired
<input type="checkbox" checked /> // not isChecked
```

Reply:
코드 내에서 사용하는 required값과 구분되어 사용하기 위해 `is`를 붙였는데 🤔  DX를 생각하면 굳이 그럴필요는 없겠군요 🙆